src/test/videopress/local-helpers/constants.js

```diff
@@ -17,6 +17,10 @@ export const PLAYBACK_SETTINGS = [
 
 export const PLAYBACK_BAR_COLOR_SETTINGS = [ 'Dynamic color' ];
 
+export const RATING_OPTIONS = [ 'G', 'PG-13', 'R' ];
+
+export const PRIVACY_OPTIONS = [ 'Site default (Public)', 'Public', 'Private' ];
```

Comment:
The first privacy option (`Site default (Public)`) can also be `Site default (Private)` if the site is private. I'm wondering if it would be worth adding a test case for this, WDYT? Note that it would require mocking the hook or the fetch request that returns the site privacy value.

Reply:
I think that's a great idea! I looked a bit into what was needed and see that it would involve mocking the `useSyncMedia` hook in order to get the `privateEnabledForSite` value, like [here](https://github.com/Automattic/jetpack/blob/3897da7aa9a4ac01cd04772ae9519903d46d60b4/projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.js#L92-L93). I think that would be easier to get working when that hook has been mocked for testing the syncing, so will make a note to follow up with a different PR after that. :)